src/dcm/agent/plugins/builtin/unmount_volume.py

```diff
@@ -48,22 +67,38 @@ def __init__(self, conf, job_id, items_map, name, arguments):
         super(UnmountVolume, self).__init__(
             conf, job_id, items_map, name, arguments)
 
-    def umount(self):
+    def unmount(self, platform):
         device_mappings = utils.get_device_mappings(self.conf)
 
+        if self.args.encrypted:
+            _g_logger.info("%s is an encrypted device...appending es" % self.args.deviceId)
+            match_device = "es" + self.args.deviceId
```

Comment:
I think we need to do this after the name translation or include it as part of the name translation.